orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBFlowStage.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+import OrientDBFlowStage._
+import akka.NotUsed
+import akka.stream.{Attributes, FlowShape, Inlet, Outlet}
+import akka.stream.stage._
+import com.orientechnologies.orient.`object`.db.OObjectDatabaseTx
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool
+import com.orientechnologies.orient.core.record.ORecord
+import com.orientechnologies.orient.core.record.impl.ODocument
+import com.orientechnologies.orient.core.tx.OTransaction
+
+import scala.collection.mutable
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+final case class IncomingMessage[T](oDocument: T)
```

Comment:
`IncomingMessage ` and `OutgoingMessage` are exactly the same. Is there any reason to have both?

Also, I think that they are switched (at least not consistent with other connectors):
The Source should emit `IncomingMessage` and the sink should ingest `OutgoingMessage`.